src/data/content-items/data-source.js

```diff
@@ -159,6 +159,18 @@ export default class ContentItem extends oldContentItem.dataSource {
     }));
   };
 
+  CORE_LIVE_CONTENT = this.LIVE_CONTENT;
+
+  LIVE_CONTENT = async () => {
+    const { Auth } = this.context.dataSources;
+    const { id: personID } = await Auth.getCurrentPerson();
+    const groupID = ROCK_MAPPINGS.CONTENT_PREVIEWS_GROUP_ID;
+    const associations = await this.request('GroupMember')
+      .filter(`GroupId eq ${groupID} and PersonId eq ${personID}`)
+      .get();
+    return associations ? null : this.CORE_LIVE_CONTENT();
```

Comment:
```suggestion
    return associations || this.CORE_LIVE_CONTENT();
```

Reply:
not sure that's the same...I don't want it it to return items from the `GroupMember` table